src/packages/cdk/src/app/api.ts

```diff
@@ -0,0 +1,104 @@
+import * as cdk from 'aws-cdk-lib';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import { NodejsFunction } from 'aws-cdk-lib/aws-lambda-nodejs';
+import { LogGroup } from 'aws-cdk-lib/aws-logs';
+import { AccessLogFormat, LambdaRestApi, LogGroupLogDestination } from 'aws-cdk-lib/aws-apigateway';
+import { Certificate } from 'aws-cdk-lib/aws-certificatemanager';
+import { Construct } from 'constructs';
+
+import { DatabaseStack } from './database';
+
+import { GraphweaverAppConfig } from './types';
+
+export class ApiStack extends cdk.Stack {
+	public readonly lambda: lambda.Function;
+
+	constructor(
+		scope: Construct,
+		id: string,
+		databaseStack: DatabaseStack,
+		config: GraphweaverAppConfig,
+		props?: cdk.StackProps
+	) {
+		super(scope, id, props);
+
+		if (!databaseStack.dbInstance.secret?.secretFullArn)
+			throw new Error('Missing required secret ARN for database');
+
+		// Create GraphQL Lambda Function
+		this.lambda = new NodejsFunction(this, `${id}-api-function`, {
+			runtime: lambda.Runtime.NODEJS_20_X,
```

Comment:
Would be nice to let them override this (optionally) in the config, e.g. `config.lambda.runtime ?? lambda.Runtime.NODEJS_20_X`